shared-bindings/usb_cdc/Serial.c

```diff
@@ -68,7 +68,8 @@
 //|         return everything readable if no newline is found and ``timeout`` is 0.
 //|         Return ``None`` in case of error.
 //|
-//|         This is a binary stream: the newline character cannot be changed.
+//|         This is a binary stream: the newline character "\\n" cannot be changed.
+//|         If the host computer transmits "\\r" it will be included as part of the line.
```

Comment:
The character in the line 72 is supposed `"\n"`, instead of the `"\r"`.

Reply:
Both are true. I meant that `\r` is not considered to be a newline, it's just another character.